### ISSUE TITLE
Still any problems when using windows for development?

### ISSUE THREAD

**Issue by hamburml** [niftylettuce/email-templates#195]
Hi!

I am using nodemailer 2.6.0 to send mails when a user registered. Now I want to use templates for the mails and localization. It's great that node-email-templates supports localization with 2.4.0 version!

I develop mainly on windows 10 x64 and deploy to linux (debian 8). The readme says that developing on windows is possible but you need to use vagrant and follow a special windows installation guide for contextify.

I searched for contextify and found out that contextify is merged into node itself (see https://github.com/brianmcd/contextify/blob/master/README.md). Does that mean that I don't need vagrant anymore and node-email-templates works out of the box on windows? After searching some more I found out that node-gyp could be a problem because it is not directly working on windows (needs some vs 2013/2015 stuff). I develop with vs 2015 with installed node.js tools so maybe that is not a big problem...

Anyways, I am not sure if I should try it because I feel a little bit lost.

Thanks for any help :)

**Comment by jeduan** [comment]
Most problems are from using compiled libraries, like `node-sass`. Node has advanced a lot on windows in the recent years. You're welcome to try and report results back.

**Comment by Michael-Hamburger** [comment]
I was able to use email-template with handlebars on win 10 x64 with node 6.5.0 without any problems. :+1: 

But I was confused because I didn't need to specify which template engine (handlebars, mustache, ejs, ...) I want to use. I just created a html.hbs (hbs for handlebars) and it worked like magic. Does email-templates uses the file-ending to distinct which template engine is used?

Thanks
